modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm

```diff
@@ -63,13 +63,17 @@
 /obj/machinery/self_actualization_device/Initialize(mapload)
 	. = ..()
 	update_appearance()
+	create_reagents(300, OPENCONTAINER) // BUBBER EDIT
```

Comment:
```suggestion
	create_reagents(600, OPENCONTAINER) // BUBBER EDIT
```
No reason to make it this restricted especially in high units of synthflesh

Reply:
I imagine you can't withdraw this?